bin/reth/src/commands/node/events.rs

```diff
@@ -357,13 +396,19 @@ where
                     );
                 }
             } else if let Some(latest_block) = this.state.latest_block {
-                info!(
-                    target: "reth::cli",
-                    connected_peers = this.state.num_connected_peers(),
-                    %freelist,
-                    %latest_block,
-                    "Status"
-                );
+                let now =
+                    SystemTime::now().duration_since(UNIX_EPOCH).unwrap_or_default().as_secs();
+                if now - this.state.latest_block_time.unwrap_or(0) > 60 {
```

Comment:
I believe this can be rewritten as
```suggestion
                if this.state.latest_block_time.map_or(true, |latest_block_time| latest_block_time.elapsed() > Duration::from_secs(60) {
```

if we also change the type of `this.state.latest_block_time` to `Option<Instant>`

Reply:
ah, we set the `this.state.latest_block_time` from `head.timestamp` which is already `u64`. Then it's fine with me 👍